spring-cloud-dataflow-server-core/pom.xml

```diff
@@ -37,6 +37,10 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-dataflow-core</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Is there an impact of using deployer local in the dataflow server core module?.

Reply:
hmm, not sure how this ended up in there - seems like it can be removed.

Reply:
Created #2639